packages/api/prisma/migrations/20230319033722_/migration.sql

```diff
@@ -0,0 +1,41 @@
+-- CreateTable
+CREATE TABLE "Book" (
+    "id" INTEGER NOT NULL,
+    "name" TEXT NOT NULL,
+    "search" TEXT NOT NULL,
+
+    CONSTRAINT "Book_pkey" PRIMARY KEY ("id")
+);
```

Comment:
This migration was generated by prisma and is ready to be deployed to a production database